scilpy/samplingscheme/gen_scheme.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import division
```

Comment:
Some future import got back on this version, Are they required ?

Reply:
weird I'm fixing this